src/diffusers/pipelines/pag/pag_utils.py

```diff
@@ -216,6 +281,19 @@ def set_pag_applied_layers(self, pag_applied_layers):
             self._check_input_pag_applied_layer(pag_layer)
 
         self.pag_applied_layers = pag_applied_layers
+        
+    def set_pag_applied_layers_sd3(self, pag_applied_layers):
```

Comment:
let's maybe make a  `pag_sd3` folder with a new`SD3PAGMixin` with a `set_pag_applied_layers`, `_set_pag_attn_processor` and `_check_input_pag_applied_layer` specific to sd3

cc @DN6 here let me know what you think!

Reply:
so we just just one "pag" folder then? ok with that